tensorflow/contrib/framework/BUILD

```diff
@@ -24,7 +27,42 @@ py_library(
         "python/ops/prettyprint_ops.py",
         "python/ops/variables.py",
     ],
+    data = [
+        ":python/ops/_variable_ops.so",
```

Comment:
`git revert` is not the problem... first I did `git fetch upstream master`, then `git rebase upstream/master` on master branch, then `git rebase origin/master` on a new branch which I cherry-pick some commits from this pr branch. test still failed.